.github/workflows/test.yml

```
@@ -13,13 +13,24 @@ on:
 
 jobs:
   test:
-    name: Linting, checks & tests
+    name: Linting, checks & tests (${{ toJSON(matrix) }})
 
     strategy:
       matrix:
-        working-directory:
-          - './packages/hidp'
-          - './project'
+        django-version: ['4.2', '5.2']
+        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12', '3.13']
+        working-directory: ['./packages/hidp']
+        exclude:
+          - django-version: '4.2'
+            python-version: '3.13'
+          - django-version: '5.2'
+            python-version: '3.8'
+          - django-version: '5.2'
+            python-version: '3.9'
```

Python 3.8 is not supported anymore: https://devguide.python.org/versions/
```suggestion
        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13']
        working-directory: ['./packages/hidp']
        exclude:
          - django-version: '4.2'
            python-version: '3.13'
          - django-version: '5.2'
            python-version: '3.9'
```